.github/workflows/ci-actions.yml

```diff
@@ -547,6 +547,11 @@ jobs:
           java-version: 11
       - name: Reclaim Disk Space
         run: .github/ci-prerequisites.sh
+      # We do this so we can get better analytics for the downloaded version of the build images
+      - name: Update Docker Client User Agent
+        shell: bash
+        run: |
+          cat <<< $(jq '.HttpHeaders += {"User-Agent": "Quarkus-CI-Docker-Client"}' ~/.docker/config.json) > ~/.docker/config.json
```

Comment:
I would add the github org/repo name into the agent (based on github actons vars) - then we can distinguish between quarkus core builds and contributors.

Reply:
I thought of that but decided against it because it would potentially make filtering harder (I don't know how filtering is done, so I wanted to be on the safe side)